.github/workflows/ci/pkg/bot/check.go

```diff
@@ -29,8 +29,15 @@ import (
 
 // Check checks if all the reviewers have approved the pull request in the current context.
 func (c *Bot) Check(ctx context.Context) error {
-	pr := c.Environment.Metadata
-	if c.Environment.IsInternal(pr.Author) {
+	author := c.Environment.Metadata.Author
+	// Assign reviewers that may have not been assigned by the assign workflow.
+	// Github requests the diff who the bot is requesting and who was already
```

Comment:
I'm not sure I understand this part of comment. Are you saying that the bot requests those reviewers that the bot requires, but have not already been requested?

Reply:
Correct. Required reviewers change based on what kind of files have changed. For example, if a pull request gets opened with no docs changes, it won't assign the required docs reviewers. If the author makes an edit to docs after the pull request opens, then it will require the docs reviewers to approve, but doesn't assign them because the `assign` workflow only requests reviewers. This change assigns them.

Reply:
I came to ask the same question - the comment is a little confusing to me. What if we said something like:

```
// Assign reviewers again, as the required reviewers may have changed since
// the initial assign workflow (due to new changes, for example)
```